lib/parser/word/entry.rb

```diff
@@ -1,22 +1,41 @@
+require 'digest'
+
 require_relative './entry/dictionary_entry'
 require_relative './entry/inflection'
 
 module Parser
   class Word
     class Entry
       def initialize(doc)
+        @content = doc.inner_html
         @dictionary_entries = doc.xpath('./dict').map { |d| DictionaryEntry.new(d) }
         @inflections = doc.xpath('./infl').map { |i| Inflection.new(i) }
       end
 
-      def to_xml
-        @to_xml ||= %(<entry>#{components.map(&:to_xml).join("\n")}</entry>\n)
+      def bamboo_xml
+        @bamboo_xml ||= <<~XML
+          <oac:hasBody rdf:resource="urn:uuid:idm#{digest}"/>
+          <oac:Body rdf:about="urn:uuid:idm#{digest}">
+            <rdf:type rdf:resource="cnt:ContentAsXML"/>
+            <cnt:rest xmlns:cnt="http://www.w3.org/2008/content#">
+              <entry uri="">
```

Comment:
The third difference is the `uri`. There is no difference for Greek but for Latin the existing service makes a call to `http://data.perseus.org/collections/` and puts something like `http://data.perseus.org/collections/urn:cite:perseus:latlexent.lex13936.1` as the `uri`.

I think we should remove the call to an external service to fetch the URI because it's slow and because when the service is down it would result in the same resource has a different result. I was thinking of doing something similar to the `rdf:resource` value above. We could take the entry and hash it or grab the headword and return `https://morph.perseids.org/analysis/word?lang=grc&engine=morpheusgrc&word=HEADWORD`.

Reply:
my feeling is that if the URIs are not actually being maintained and supported as persistent identifiers for the data object they are identifying, then they don't really any value here.  This was an optimistic feature of the old morphology service -- i.e. we thought we would eventually have persistent URIs for all of the greek and latin forms and lemmas via a lexical inventory that could be collaboratively maintained.  It's still a desirable goal, but it's not something that can be solved by individual projects.  The community needs to come together on this for the uris to have any meaning or longevity.  The LiLA project is trying to do this for Latin, I think it remains to be seen if it will be successful. They are minting URIs for lemmas but not all forms, I don't think. At least not yet.  Part of making it successful is people actually using the URIs, and to that point, I could see value in trying to use them here, but that would be on the headword and not the entry, and it almost certainly could not be a runtime query that retrieved them but would have to be something that was mapped locally with the morpheus data.

Reply:
Thanks! It sounds like a good solution for now is to leave the field blank but to pay attention to LiLA and possibly incorporate their URI system later.